eclass/cargo.eclass

```diff
@@ -12,12 +12,8 @@
 if [[ -z ${_CARGO_ECLASS} ]]; then
 _CARGO_ECLASS=1
 
-if [[ ${PV} == *9999* ]]; then
-	# we need at least this for cargo vendor subommand
-	CARGO_DEPEND=">=virtual/cargo-1.37.0"
-else
-	CARGO_DEPEND="virtual/cargo"
-fi
+# we need this for 'cargo vendor' subcommand and net.offline config knob
+CARGO_DEPEND=">=virtual/cargo-1.37.0"
```

Comment:
isn't virtual/cargo supposed to go away?

Reply:
It can't be easily eliminated, because removing this dep from the eclass doesn't remove it from the users installed package database.

https://archives.gentoo.org/gentoo-dev/message/0060e04a57f82ba7d214bdfea64f4afc